GenderPayGap.WebUI/wwwroot/styles/components/_accessible-autocomplete.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../assets/stylesheets/accessible-autocomplete.min.css";
+
+.autocomplete-wrapper {
+  max-width: 20em;
+  margin-bottom: 4rem;
```

Comment:
Where do these measurements come from?
It's odd to use both `em` and `rem`.

I'm not sure what the `govuk-*` CSS uses.
We might even be able to extend / include some parts of that (e.g. could we `@extend govuk-input--width-20`?)

Note:
> I just looked up what units `govuk-input--width-20` uses.
> It's... `ex`
> Wait, what?
> 
> Also, I didn't realise quite how many units of length browsers support!
> https://developer.mozilla.org/en-US/docs/Web/CSS/length#ex

Reply:
These were also copied from somewhere, but are not needed for what we need

Reply:
I've gone through now and pulled out a few more css bits that were also unnecessary